src/components/DateRangePicker.jsx

```diff
@@ -198,6 +199,14 @@ export default class DateRangePicker extends React.Component {
     }
   }
 
+  getDateString(date) {
+    const { displayFormat } = this.props;
+    if (displayFormat) {
+      return date && date.format(displayFormat);
```

Comment:
would it be better to do `if (date && displayFormat)` so that falsy dates always go through `toLocalizedDateString`? With the current approach, passing `NaN` with a format will return `NaN`, for example.